contracts/wrapped_azero/lib.rs

```diff
@@ -129,31 +125,32 @@ pub mod wrapped_azero {
         /// Will revert if called from an account without a proper role
         #[ink(constructor)]
         pub fn new() -> Self {
-            let caller = Self::env().caller();
-            let code_hash = Self::env()
-                .own_code_hash()
-                .expect("Called new on a contract with no code hash");
-
-            let access_control = AccountId::from(ACCESS_CONTROL_PUBKEY);
-            let access_control = AccessControlRef::from_account_id(access_control);
-            if access_control.has_role(caller, Role::Initializer(code_hash)) {
-                let metadata = metadata::Data {
-                    name: Some("wAzero".into()),
-                    symbol: Some("wA0".into()),
-                    decimals: 12, // same as AZERO
-                    ..Default::default()
-                };
-
-                Self {
-                    psp22: psp22::Data::default(),
-                    metadata,
-                    access_control,
-                }
-            } else {
-                panic!("Caller is not allowed to initialize this contract");
+            let metadata = metadata::Data {
+                name: Some("wAzero".into()),
+                symbol: Some("wA0".into()),
+                decimals: 12, // same as AZERO
+                ..Default::default()
+            };
+
+            Self {
+                psp22: psp22::Data::default(),
+                metadata,
             }
         }
 
+        /// Terminates the contract.
+        ///
+        /// No-op by default, can only be compiled with a flag in dev environments
+        #[ink(message)]
+        pub fn terminate(&mut self) -> Result<()> {
+            cfg_if::cfg_if! { if #[cfg( feature = "devnet" )] {
```

Comment:
simple `#[cfg(feature = "devnet")]` doesn't work?

Reply:
noup. I might revisit it - but in another PR, this not part of the ticket